src/AdminApi/ScriptTag/Builders/CountScriptTagsRequest.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Yaspa\AdminApi\ScriptTag\Builders;
+
+use DateTime;
+use Yaspa\AdminApi\ScriptTag\Constants\ScriptTag;
+use Yaspa\Constants\RequestBuilder;
+use Yaspa\Traits\AuthorizedRequestBuilderTrait;
+
+/**
+ * Class CountScriptTagsRequest
+ *
+ * @package Yaspa\AdminApi\ScriptTag\Builders
+ * @see https://help.shopify.com/api/reference/scripttag#count
+ */
+class CountScriptTagsRequest
+{
+    use AuthorizedRequestBuilderTrait;
+
+    const URI_TEMPLATE = 'https://%s.myshopify.com/admin/script_tags/count.json';
+
+    /** @var string $src */
+    protected $src;
+
+    /**
+     * CountScriptTagsRequest constructor.
+     */
+    public function __construct()
+    {
+        // Set properties with defaults
+        $this->uriTemplate = self::URI_TEMPLATE;
+        $this->httpMethod = RequestBuilder::GET_HTTP_METHOD;
+        $this->headers = RequestBuilder::JSON_HEADERS;
+        $this->bodyType = RequestBuilder::QUERY_BODY_TYPE;
+    }
+
+    /**
+     * @return array
+     */
+    public function toArray(): array
+    {
+        $array = [];
+
+        if (!empty($this->ids)) {
```

Comment:
Does count support `ids` parameter? Docs seem to show only the ability to get all script tags for a shop or "Count script tags with given URL"?